mne_bids/write.py

```diff
@@ -485,6 +502,14 @@ def _scans_tsv(raw, raw_fname, fname, overwrite=False):
             raise FileExistsError(f'"{raw_fname}" already exists in '
                                   f'the scans list. Please set '
                                   f'overwrite to True.')
+
+        for key in data.keys():
+            if key in orig_data:
+                continue
+
+            # add 'n/a' if any missing columns
+            orig_data[key] = ['n/a'] * len(next(iter(data.values())))
```

Comment:
I'm not sure what you're intending to do here, could this be solved via 

`.... len(list(data.values()))`
?

Reply:
This doesn't actually work because we want the length of the number of rows there are, not the number of values. See `_combine_rows` logic.

Reply:
I'm not sure I understand but I'll just trust you 😁

Reply:
hehe so len(list(data.values())), will give a list of N ['n/a'], where N is the number of columns.

We want N to be the number of rows. Since we're not using pandas, this was the shortest 1-liner I found :p.

Reply:
Urrghhs… good it works 🙈 thanks for the explanation